contrib/jemalloc/src/jemalloc.c

```diff
@@ -2366,7 +2366,9 @@ JEMALLOC_EXPORT JEMALLOC_ALLOCATOR JEMALLOC_RESTRICT_RETURN
 void JEMALLOC_NOTHROW *
 JEMALLOC_ATTR(malloc) JEMALLOC_ALLOC_SIZE(1)
 je_malloc(size_t size) {
-	bool zero_size;
+#ifdef __CHERI_PURE_CAPABILITY__
+	bool zero_size = (size == 0);
+#endif
```

Comment:
This variable's existence is a bug. I'll try to take another look at jemalloc soon to remove this diff.